test/Common.FunctionalTests/Utilities/FunctionalTestsBase.cs

```
@@ -25,14 +26,7 @@ public FunctionalTestsBase(ITestOutputHelper output = null) : base(output)
             }
 
             // Currently hosting throws if the Servertype = IIS.
-            if (parameters.ServerType == ServerType.IIS)
-            {
-                _deployer = new IISDeployer(parameters, LoggerFactory);
-            }
-            else
-            {
-                _deployer = ApplicationDeployerFactory.Create(parameters, LoggerFactory);
-            }
+            _deployer = DeployerSelector.GetDeployer(parameters, LoggerFactory);
```

Are we just silently ignoring parameters.Server?

---

Yeah, we have to if we wanted minimal code change between the two projects. I can remove setting .Server.

---

Hm, what about exposing current server type as a static?

---

Sure, though I think we need to do something more. Is there a clean way to modify the test name to append the server type?

---

Not that I know of.